defaults/keypath/keypath.go

```diff
@@ -1,14 +1,14 @@
 package keypath
 
 const (
-	AirgapKeyPath     = "/go/src/github.com/rancher/tfp-automation/modules/airgap"
-	AirgapRKE2KeyPath = "/go/src/github.com/rancher/tfp-automation/modules/airgapRKE2"
-	K3sKeyPath        = "/go/src/github.com/rancher/tfp-automation/modules/k3s"
-	ProxyKeyPath      = "/go/src/github.com/rancher/tfp-automation/modules/proxy"
-	RegistryKeyPath   = "/go/src/github.com/rancher/tfp-automation/modules/registries"
-	RKEKeyPath        = "/go/src/github.com/rancher/tfp-automation/modules/rke"
-	RKE2KeyPath       = "/go/src/github.com/rancher/tfp-automation/modules/rke2"
-	RancherKeyPath    = "/go/src/github.com/rancher/tfp-automation/modules/rancher2"
-	SanityKeyPath     = "/go/src/github.com/rancher/tfp-automation/modules/sanity"
-	UpgradeKeyPath    = "/go/src/github.com/rancher/tfp-automation/modules/upgrade"
+	AirgapKeyPath     = "/src/github.com/rancher/tfp-automation/modules/airgap"
```

Comment:
Why are the keypaths being edited here? In the Linode PR, this was left alone and it worked fine, this will break the tests I would think.

Reply:
there was a miss in the previous PR for updating the go path. This is tied to the change in framework/set/resources/rancher2/setKeyPath.go iirc